collectors/node.go

```diff
@@ -27,6 +29,8 @@ import (
 )
 
 var (
+	resyncPeriod = 5 * time.Minute
```

Comment:
As `resyncPeriod` is referenced in all go files under `controllers` package, i think it is better to put it into a single file, such as `controllers.go` under `controllers` package. :)